### ISSUE TITLE
[BUG] Games not visible in VR headset (but SteamVR Home is fine), also often a second copy of Steam starts

### ISSUE THREAD

**Issue by SentineI** [ValveSoftware/SteamVR-for-Linux#595]
**Describe the bug**
When I start SteamVR, it seems fine in SteamVR Home, but when I launch any VR game it stays stuck on the loading screen - but on the computer monitor I can usually see the game in a window.  i.e. The game starts, but it's not directed to my VR headset.

I also noticed that starting a VR game often seems to cause the current Steam to get disconnected, and a second copy of Steam to start.  This seems to be due to some kind of Steam updating message when I start a Steam VR game.

**To Reproduce**
1. Start SteamVR (BETA).
2. Put on headset to use SteamVR Home.
3. Start a VR game, e.g. Walkabout Mini Golf, e.g. Karnage Chronicles.

**Expected behavior**
There is the VR loading screen, and then the game starts.  And Steam does not start a second copy of itself.

**System Information (please complete the following information):**
* Distribution: Ubuntu LTS 22.04.2
* SteamVR version: 1.26.6
* Steam client version: 1689034492
* Opted into Steam client beta?: No
* Graphics driver version: NVidia 535.54.03
* Gist for SteamVR System Information: https://gist.github.com/SentineI/3077ef8858ef018cbab04b5199bfa5d1

**Screenshots**
n/a

**Additional context**
I also get bug #577, but I'm not sure if it's related.  
SteamVR *used* to work fine on my system, but I have updated Ubuntu & especially the NVidia driver a fair bit since then.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by ncointe** [comment]
Same issue here, plus as a Godot Engine Apps developper I have an error message indicating that OpenXR interface is not found when trying to start any app.

My VR home says I do not have any game to play (which is wrong) and  I also see blank windows.

**System Information**

- Distribution : Archlinux
- SteamVR version : 1.26.6
- Steam client version : 1689983533
- Opted into Steam client beta?: No
- Graphics driver version: NVIDIA 535.86.05

**Comment by SentineI** [comment]
Problem still happening with:

**System Information**
* Distribution: Ubuntu LTS 22.04.3
* SteamVR version: 1.26.7
* Steam client version: 1690583737
* Opted into Steam client beta?: No
* Graphics driver version: NVidia 535.86.05

**Comment by SentineI** [comment]
Also no better when trying the beta version of Steam VR:
**System Information**
* SteamVR version: 1.27.3
* Steam client version: 1690583737

**Comment by SentineI** [comment]
Also no better using beta version of Steam:
**System Information**
* SteamVR version: 1.27.3
* Steam client version: 1692911603
* Opted into Steam client beta?: Yes

**Comment by SentineI** [comment]
Also no better using NVIDIA 470.199.02 nor 525.x (using non-beta SteamVR & Steam client).

I also tried changing the version of Proton (since the problem seems may be limited to Windows VR games), 
but Proton 7.0-6 and 6.3-8 didn't help, and when I applied that to the SteamVR tool itself, it seemed to break SteamVR.

So my suspicion is that the problem is either due to newer versions of the Steam client or SteamVR, but I have no way of trying older versions.

Well, I *did* try opting into the SteamVR beta labelled "linux_v1.14 - Older build, for linux users" (which may be from late 2020?), but that didn't seem to help, maybe it is too old?  Although it might suggest the problem is with the Steam client itself (which did get a huge update/rewrite in recent times).

I have run out of idea to try :-(

**Comment by SentineI** [comment]
I've eventually worked out the cause of problem:  While I was using the native Steam client, I did also have the Flatpak version installed (com.valvesoftware.Steam), _and it seems that (for some strange reason) this causes the native Steam client to try to launch VR games using the Flatpak version_ (which then failed).

This was further complicated by the fact that my Flatpak version had the same games installed as the native version, since I'd effectively used "mount --bind $HOME/.steam $HOME/.var/app/com.valvesoftware.Steam/.steam".  (I have however confirmed this did not cause my problem, as part of my debugging I installed a clean Ubuntu 22.04 on a spare HD, without using that mounting trick.)

So the solution is simple if slightly annoying:  Uninstall the Flatpak version of Steam.  I'll see if I can report thus as a bug on the Flatpak version of Steam.
